lib/stencil-bundle.js

```diff
@@ -122,7 +122,7 @@ Bundle.prototype.assembleTemplatesTask = function (callback) {
             return upath.toUnix(file.replace(self.templatesBasePath + Path.sep, '').replace('.html', ''));
         });
 
-        Async.map(files, TemplateAssembler.assemble, function (err, results) {
+        Async.map(files, TemplateAssembler.assemble.bind(this, {bundle: true}), function (err, results) {
```

Comment:
This does not look too clean. I would prefer to have to methods, `TemplateAssembler.assemble` and TemplateAssembler.assembleAndBundle`